docs/get-started/getting-started-aws-emr.md

```diff
@@ -0,0 +1,132 @@
+# Get Started with Rapids on AWS EMR
+
+This is a getting started guide for Rapids on AWS EMR. At the end of this guide, the user will be able to run a sample Apache Spark application that runs on NVIDIA GPUs on AWS EMR.
+
+The current EMR 6.2.0 release support Spark version 3.0.1 and Nvidia spark-rapids 0.2.0. For more details of supported applications, please see the [EMR release notes](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-release-6x.html).  
+
+For more information on AWS EMR, please see the [AWS documentation](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-what-is-emr.html).
+
+## Configure and Launch AWS EMR with GPU Nodes
+
+###  Launch EMR Cluster using AWS CLI
+
+Following steps are based on AWS EMR document - ["Using the Nvidia Spark-RAPIDS Accelerator for Spark"](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-spark-rapids.html) 
+
+
+You can use AWS CLI to launch a cluster with one Master node (m5.xlarge) and two nodes 2x g4dn.2xlarge  
+
+```
+ aws emr create-cluster \
+--release-label emr-6.2.0 \
+--applications Name=Hadoop Name=Spark Name=Livy Name=JupyterEnterpriseGateway \
+--service-role EMR_DefaultRole \
+--ec2-attributes KeyName=my-key-pair,InstanceProfile=EMR_EC2_DefaultRole \
+--instance-groups InstanceGroupType=MASTER,InstanceCount=1,InstanceType=m4.4xlarge \                 
+                  InstanceGroupType=CORE,InstanceCount=1,InstanceType=g4dn.2xlarge \    
+                  InstanceGroupType=TASK,InstanceCount=1,InstanceType=g4dn.2xlarge \
+--configurations file:///my-configurations.json \
+--bootstrap-actions Name='My Spark Rapids Bootstrap action',Path=s3://my-bucket/my-bootstrap-action.sh
+```
+
+Please fill with actual value for KeyName and file paths. You can further customize  SubnetId, EmrManagedSlaveSecurityGroup, EmrManagedMasterSecurityGroup, name and region etc. 
+
+
+###  Launch EMR Cluster using AWS Console (GUI)
+
+Go to the AWS Management Console and select the `EMR` service from the "Analytics" section. Choose the region you want to launch your cluster in, e.g. US West Oregon, using the dropdown menu in the top right corner. Click `Create cluster` and select `Go to advanced options`, which will bring up a detailed cluster configuration page.
+
+#### Step 1:  Software, Configuration and Steps
+
+Select **emr-6.2.0** or latest EMR version for the release, uncheck all the software options, and then check **Hadoop 3.2.1**, **Spark 3.0.1**, **Livy 0.7.0** and **JupyterEnterpriseGateway 2.1.0**.
+
+In the "Edit software settings" field, copy and paste the configuration from the [EMR document](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-spark-rapids.html). You can also create a JSON file on you own S3 bucket.
+
+![Step 1: Step 1:  Software, Configuration and Steps](pics/Rapids_EMR_GUI_1.PNG)

```

Comment:
Elsewhere in the `docs/` directory we're using `img` as the directory name to hold images.  It would be nice to keep this consistent.  Also RAPIDS should be all caps in the filenames.  Interesting it's the _only_ part of the image filenames that is not capitalized. :smile:

Reply:
@mgzhao Tom pointed out that the config.json didn't have the right settings. I suggest that we provide a sample config.json here. For example; in this cluster you chose g4dn.2xlarge, which has 8 vCPU cores. The right config would be use all the cores, we need to spark.executor.cores=8, spark.task.resource.gpu.amount=0.125.
Here is the sample config for this cluster:
   {
      "Classification":"spark-defaults",
      "Properties":{
     	"spark.plugins":"com.nvidia.spark.SQLPlugin", 
     	"spark.sql.sources.useV1SourceList":"",
     	"spark.executor.extraJavaOptions":"-Dai.rapids.cudf.prefer-pinned=true",
     	"spark.executor.resource.gpu.discoveryScript":"/usr/lib/spark/scripts/gpu/getGpusResources.sh",      	
     	"spark.submit.pyFiles":"/usr/lib/spark/jars/xgboost4j-spark_3.0-1.0.0-0.2.0.jar",
     	"spark.executor.extraLibraryPath":"/usr/local/cuda/targets/x86_64-linux/lib:/usr/local/cuda/extras/CUPTI/lib64:/usr/local/cuda/compat/lib:/usr/local/cuda/lib:/usr/local/cuda/lib64:/usr/lib/hadoop/lib/native:/usr/lib/hadoop-lzo/lib/native:/docker/usr/lib/hadoop/lib/native:/docker/usr/lib/hadoop-lzo/lib/native",
     	"spark.rapids.sql.concurrentGpuTasks":"4",
     	"spark.executor.resource.gpu.amount":"1",
     	"spark.executor.cores":"8",
     	"spark.task.cpus ":"1",
     	"spark.task.resource.gpu.amount":"0.125",
     	"spark.rapids.memory.pinnedPool.size":"2G",
     	"spark.executor.memoryOverhead":"2G",
     	"spark.locality.wait":"0s",
     	"spark.sql.shuffle.partitions":"200",
     	"spark.sql.files.maxPartitionBytes":"256m",
      "spark.sql.adaptive.enabled":"false"
  	}

Reply:
I updated the FAQ.md and created subdirectories in demos and img for different CSPs.  I also updated the file names.  I believe Kong had already updated the config settings in the json.